TYPO3.Neos/Classes/TYPO3/Neos/ViewHelpers/Link/UriViewHelper.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace TYPO3\Neos\ViewHelpers\Link;
+
+/*                                                                        *
+ * This script belongs to the TYPO3 Flow package "TYPO3.Neos".            *
+ *                                                                        *
+ * It is free software; you can redistribute it and/or modify it under    *
+ * the terms of the GNU General Public License, either version 3 of the   *
+ * License, or (at your option) any later version.                        *
+ *                                                                        *
+ * The TYPO3 project - inspiring people to share!                         *
+ *                                                                        */
+
+use TYPO3\Fluid\Core\ViewHelper\AbstractTagBasedViewHelper;
+
+/**
+ * A view helper for creating links to uri's. By default the target
+ * is _blank so the links are opening in a new window.
+ * = Examples =
+ * <code title="Simple external uri">
+ * <neos:link.uri uri="http://foobar.com">Foo bar</neos:link.uri>
+ * </code>
+ * <output>
+ * <a href="http://foobar.com" target="_blank">Foo bar</a>
+ * </output>
+ * <code title="Simple internal uri">
+ * <neos:link.uri uri="http://foobar.com" target="null">Foo bar</neos:link.uri>
+ * </code>
+ * <output>
+ * <a href="http://foobar.com">Foo bar</a>
+ * </output>
+ *
+ * @api
+ */
+class UriViewHelper extends AbstractTagBasedViewHelper
```

Comment:
I think this view helper needs to be discussed because it doesn't bring much value